Build/test.ps1

```diff
@@ -38,10 +38,29 @@ function Validate-Integrals {
     }
 }
 
+function Test-AutoSubstitution {
+    Write-Host "##[info]Testing AutoSubstitution sample..."
+
+    $output = dotnet run --project (Join-Path $PSScriptRoot "../samples/runtime/autosubstitution")
+
+    if ($output -notmatch "^Quantum version\s*$") {
+        Write-Host "##vso[task.logissue type=error;]Auto substitution with QuantumSimulator failed"
```

Comment:
Can we add a bit more detail to the error log as to what the failure is? E.g.: make the message say that the wrong version of the operation was executed?